src/index.html

```diff
@@ -17,11 +17,423 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/styles.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="LogoMoyo"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav_item">
```